doc/en/user/source/community/vector-mosaic/configuration.rst

```diff
@@ -0,0 +1,37 @@
+.. _community_vector_mosaic_config:
+
+Vector Mosaic Datastore configuration
+=====================================
+
+When the extension has been installed, `Vector Mosaic Data Store` will be an option in the `Vector Data Sources` list when creating a new data store.
+
+.. figure:: images/vector-mosaic-vector-create.png
+   :align: center
+
+   *Vector Mosaic Data Store in the list of vector data sources*
+
+.. figure:: images/vector-mosaic-vector-configure.png
```

Comment:
Outdated screenshot? I don't see drop downs.

Reply:
Also, looking at the current panel:

![image](https://user-images.githubusercontent.com/325433/191261441-9fb5a67f-4ef8-46c9-95ef-eba43e44b38d.png)

- The second drop-down is not auto-fill type
- The contents inside the second drop-down should be alphabetically sorted, helps finding stuff in it when there are many stores.

Reply:
Fixed